docs/2.3. APIs - Server Side Implementation Notes.md

```diff
@@ -8,9 +8,9 @@ In order to permit web-based control interfaces to be hosted remotely, all NMOS
 
 In addition to this, where highlighted in the API specifications servers MUST respond to HTTP pre-flight OPTIONS requests. Servers MAY additionally support HTTP OPTIONS requests made to any other API resource.
 
-In order to simplify development, the following headers may be returned in order to remove these restrictions as far as possible. Note that these are very relaxed and may not be suitable for a production deployment.
+In order to simplify development, the following headers MAY be returned in order to remove these restrictions as far as possible. Note that these are very relaxed and possibly not suitable for a production deployment.
 
-```
+```json
```

Comment:
This example isn't JSON. I think this would work better...

```suggestion
```http
```

Reply:
Yes you're right